src/Makefile.am

```diff
@@ -4,14 +4,28 @@ AM_CFLAGS =
 bin_PROGRAMS = gregorio
 gregorio_SOURCES = characters.c gregorio-utils.c messages.c struct.c \
 				   unicode.c struct.h messages.h unicode.h characters.h \
-				   plugins.h dump/dump.c dump/dump.h gregoriotex/gregoriotex-write.c \
-				   gregoriotex/gregoriotex.h
+				   plugins.h config.h dump/dump.c dump/dump.h \
+				   gregoriotex/gregoriotex-write.c gregoriotex/gregoriotex.h
 
+@MK@ifneq ($(wildcard ../.git),)
+@MK@  _branch_ = $(shell git rev-parse --abbrev-ref HEAD)
+@MK@  ifneq ($(_branch_),master)
```

Comment:
I think it would be good to detect tags too, for them we just want the normal version... You can check if `git describe --exact-match HEAD` fails or not for this